src/API/Reports/Controller.php

```diff
@@ -185,6 +185,10 @@ public function get_items( $request ) {
 	public function get_order_number( $order_id ) {
 		$order = wc_get_order( $order_id );
 
+		if ( ! $order instanceof \WC_Order ) {
+			return '';
```

Comment:
Minor nitpick: I think `null` might make checks here a little easier if we need them in the future.

Reply:
Nice, done in 2707160